### PR TITLE
fix(content): replace lockwise section with section about relay

### PR DESCRIPTION
**Pull request by dschom**
## Because

- Lockwise needs to be phased out and private relay should be promoted.

## This pull request

- Removes the lockwise section underneath ‘One account, many services‘ on the enter your email splash screen.
- Phases out conditional rendering of services based on where the user is coming from.

## Issue that this pull request solves

Closes: #11031

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

image

### CODE REVIEW

File: packages/fxa-content-server/app/images/graphic-logo-relay.svg [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<svg width="434" height="80" viewBox="0 0 195 43" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (vbudhram) [inline packages/fxa-content-server/app/images/graphic-logo-relay.svg:1]: Not sure where you got graphic, but I usually like to run it through https://jakearchibald.github.io/svgomg/ to get an optimized image.

File: packages/fxa-content-server/app/scripts/templates/partial/firefox-family-services.mustache [hunk 2]
```diff
@@ -5,30 +5,20 @@
...
+      <span class="logo logo-relay">{{#t}}Firefox Relay{{/t}}</span>
+      {{#t}}Sign up for new accounts without handing over your email address.{{/t}}
+    </li>
   </ul>
-  {{#isInPocketMigration}}
-    <a class="see-all-services" href="https://www.mozilla.org/firefox/products/" rel="noopener noreferrer">
-      {{#t}}See all{{/t}}
-    </a>
-  {{/isInPocketMigration}}
+  <a class="see-all-services" href="https://www.mozilla.org/firefox/products/" rel="noopener noreferrer">
```
Comment (vbudhram) [inline packages/fxa-content-server/app/scripts/templates/partial/firefox-family-services.mustache:21]: 👍🏽

**Review by vbudhram — APPROVED** [review]
@dschom Thanks! LGTM

File: packages/fxa-content-server/app/images/graphic-logo-relay.svg (hunk 1 above)
Comment (dschom) [reply packages/fxa-content-server/app/images/graphic-logo-relay.svg:1]: The graphic was supplied in jira.

Comment (annasob) [reply packages/fxa-content-server/app/images/graphic-logo-relay.svg:1]: i got the graphic from UX and added it to the jira. just fyi

Comment (dschom) [reply packages/fxa-content-server/app/images/graphic-logo-relay.svg]: Alright, the graphic has been optimized.

**Pull request merged**